Fixed bingo SQL installl scripts path after #50

Lucas-Gluchowski opened the pull request:
#50 introduced Postgres versioning of build but installation SQL scripts hasn't been changed.
I checked it on Mac OS X, so I might be worth checking it on Windows.

AlexanderSavelyev commented:
@Lucas-Gluchowski did you specify environment variables?
BINGO_PG_DIR and BINGO_PG_VERSION 

as it is required here
https://github.com/epam/Indigo/blob/master/bingo/postgres/README-BUILD.md

Lucas-Gluchowski commented:
@AlexanderSavelyev I have added env variables BINGO_PG_DIR and BINGO_PG_VERSION but only when I added /common/ to path building script worked.Take a look into directory structure (https://github.com/epam/Indigo/tree/master/bingo/postgres/sql) there are 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 and common directories but last is not included in path building.

AlexanderSavelyev commented:
@Lucas-Gluchowski please check the "dist" folder after the build procedure is finished. It should contain an archive, e.g. bingo-postgres9.4-1.7.10-dev7-linux64.zip
This a deliverable bingo cartridge, which should be used to install for end user. The idea is that sql folder inside does not contain "common" or any other subfolders. In other words, you should not use the "bingo/postgres/bingo-pg-install.sh" script directly

Lucas-Gluchowski commented:
Looks like I misuse installation scripts. I didn't use final cartridge, but produced SQL scripts.
